### PR TITLE
Remove unnecessary comments and fix logical error

### PR DESCRIPTION
**Pull request by Andrewmob2703-zz**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -61,7 +61,7 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     // Get BlogMessages from Datastore.
-    List<BlogMessage> blogMessages = LoadAllBlogsOrLast(true);
+    List<BlogMessage> blogMessages = LoadAllBlogsOrLast(/**all=*/ true);
```
Comment (peizhaoou) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: For nested comment, you would probably want to just use one star like /*all=*/

**Review by peizhaoou — APPROVED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (martiKallas) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Looks like the markup text is formatting Peizhao's comment maybe like:
/\*all=\*/

**Review by martiKallas — APPROVED** [review]
Great use of a small PR for a quick fix!

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (peizhaoou) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Thanks Marti. Yup, I didn't realize that.

**Pull request merged**